client/app/components/Paginator.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { react2angular } from 'react2angular';
+import AntPagination from 'antd/lib/pagination';
+
+export function Pagination({
+  page,
+  itemsPerPage,
+  totalCount,
+  onChange,
+}) {
+  if (totalCount <= itemsPerPage) {
+    return null;
+  }
+  return (
+    <div className="paginator-container">
+      <AntPagination
+        className="pagination"
+        defaultCurrent={page}
+        defaultPageSize={itemsPerPage}
+        total={totalCount}
+        onChange={onChange}
+      />
+    </div>
+  );
+}
+
+Pagination.propTypes = {
+  page: PropTypes.number.isRequired,
+  itemsPerPage: PropTypes.number.isRequired,
+  totalCount: PropTypes.number.isRequired,
+  onChange: PropTypes.func,
```

Comment:
Any reason not to require `onChange` unless we have a need for a "view only" mode?

Reply:
Well, I don't know. Currently we use this handler everywhere, but unlike other properties, this one is not required to render the component, so I decided to make it optional.